### PR TITLE
Homework7: Sergey Streltsov

### PR DESCRIPTION
**Pull request by Mantikor**

### CODE REVIEW

File: src/homework07.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    Include else and finally clauses in your try-except block.
+    """
+    try:
+        l = thedict[listname]
+    except KeyError as e:
+        thedict[listname] = []
+        print("Created %s." % listname)
+    else:
+        print("%s already has %d elements." % (listname, len(l)))
+    finally:
```
Comment (belovmd) [inline src/homework07.py:122]: we can not to use finally here and just put it to function

**Review by belovmd — COMMENTED** [review]
I really like your tests! good job! 
I've found just one little issue:

File: src/homework07.py (hunk 1 above)
Comment (Mantikor) [reply src/homework07.py:122]: i don't understood about finally, by condition of the task, we need to use: Include else and finally clauses in your try-except block

**Review by belovmd — COMMENTED** [review]
9,5

**Pull request closed without merging**